src/app/layout.tsx

```diff
@@ -20,16 +23,22 @@ export const metadata: Metadata = {
     'Yiqi is a platform for bringing people together through professional communities. Find your tribe, learn, grow, and connect.'
 }
 
-export default function RootLayout({
+export default async function RootLayout({
   children
 }: Readonly<{
   children: React.ReactNode
 }>) {
+  const user = await getUser()
+
   return (
     <html lang="en" className="h-screen w-full">
       <body
         className={`${geistSans.variable} ${geistMono.variable} antialiased h-screen w-full`}
       >
+        <MainLandingNav
+          user={{ name: user?.name, picture: user?.picture as string }}
```

Comment:
we should not need  `as ` keyword... if the type is wrong then we need to fix this